client/src/lib/components/Cards/CollectionCards/CollectionFullCard.tsx

```diff
@@ -4,26 +4,53 @@ import styled from 'styled-components/native';
 
 import {COLORS} from '../../../../../../shared/src/constants/colors';
 import Image from '../../Image/Image';
-import {Display20} from '../../Typography/Display/Display';
+import {Display22} from '../../Typography/Display/Display';
 import {SPACINGS} from '../../../constants/spacings';
 import TouchableOpacity from '../../TouchableOpacity/TouchableOpacity';
-import {Spacer4, Spacer8} from '../../Spacers/Spacer';
+import {Spacer16, Spacer4, Spacer8} from '../../Spacers/Spacer';
 import SessionProgress from '../../SessionProgress/SessionProgress';
+import {CollectionIcon} from '../../Icons';
+import {Body12} from '../../Typography/Body/Body';
+import {PlayfairDisplayMedium} from '../../../constants/fonts';
+import LinearGradient from 'react-native-linear-gradient';
 
-export const HEIGHT = 174;
+export const HEIGHT = 114;
 
 type CollectionFullCardProps = {
-  title?: string;
+  title: string;
+  description?: string;
   image: ImageSourcePropType;
   progressItems: Array<boolean>;
+  theme?: {backgroundColorGradient?: string[]; textColor?: string};
   onPress: () => void;
 };
 
 const Container = styled(TouchableOpacity)({
   height: HEIGHT,
   backgroundColor: COLORS.GREYLIGHTEST,
-  paddingVertical: SPACINGS.FOUR,
-  paddingHorizontal: SPACINGS.SIXTEEN,
+  borderRadius: SPACINGS.SIXTEEN,
+});
+
+const Heading = styled(Display22)<{color?: string}>(
+  ({color = COLORS.BLACK}) => ({
+    fontFamily: PlayfairDisplayMedium,
+    lineHeight: 27,
+    color,
+  }),
+);
+
+const Description = styled(Body12)<{color?: string}>(
+  ({color = COLORS.BLACK}) => ({
+    fontWeight: 500,
```

Comment:
I don't think setting fontWeight works as expected on both iOS and Android. If I remember correctly yoou need to use the correct weighted font. Like the above `PlayfairDisplayMedium`